src/ui/MLFacet/MLFacet.ts

```diff
@@ -292,9 +313,9 @@ export class MLFacet extends Component {
       return this.notImplementedError();
     }
 
-    const facetResponse = findWhere(data.results.facets, { field: this.fieldName });
+    const { values = [], moreValuesAvailable = false } = findWhere(data.results.facets, { field: this.fieldName });
 
-    this.onQueryResponse(facetResponse ? facetResponse.values : []);
+    this.onQueryResponse(values, moreValuesAvailable);
```

Comment:
I think it is nicer to pass the entire response object, and deconstruct inside the `onQueryResponse` rather than deconstructing outside, and adding a second argument.